File: DVLD_Project/Form13.vb
Imports BussnessLayerForDVLD
Imports System
Imports System.Windows.Forms

Namespace DVLD_Project
    Public Partial Class Form13
        Inherits Form
        Private _I As Integer = 0
        Private _N As String = "", _O As String = ""
        Private _F As Double = 0.0
        Public Sub New()
            InitializeComponent()
        End Sub

        Public Sub New(I As Integer, N As String, O As String, F As Double)
            InitializeComponent()
            _I = I
            _N = N
            _O = O
            _F = F
            lblTypeID.Text = I.ToString()
            tbTestName.Text = N.ToString()
            tbNote.Text = O.ToString()
            tbFees.Text = F.ToString()
        End Sub

        Private Sub guna2Button1_Click(sender As Object, e As EventArgs)
            clsBussnesLayer.UpdateTestType(_I, tbTestName.Text, tbNote.Text, Convert.ToDouble(tbFees.Text))
        End Sub
    End Class
End Namespace

File: DVLD_Project/Form13.cs
using BussnessLayerForDVLD;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace DVLD_Project
{
    public partial class Form13 : Form
    {
        int _I = 0;
        string _N = "", _O = "";
        double _F = 0.0;
        public Form13()
        {
            InitializeComponent();
        }

        public Form13(int I,string N,string O,double F)
        {
            InitializeComponent();
            _I = I;
            _N = N;
            _O = O;
            _F = F;
            lblTypeID.Text=I.ToString();
            tbTestName.Text=N.ToString();
            tbNote.Text=O.ToString();
            tbFees.Text=F.ToString();
        }

        private void guna2Button1_Click(object sender, EventArgs e)
        {
            clsBussnesLayer.UpdateTestType(_I, tbTestName.Text, tbNote.Text,Convert.ToDouble( tbFees.Text));
        }
    }
}
